site/source/tutorials/006_deploying_to_production/index.md

```diff
@@ -0,0 +1,143 @@
+
+In addition to the files, the `dist` folder contains three subdirectories. Two of those, `images` and `styles` are copies from the `src` directory. The third directory, `src`, contains two files - `main.js` and `main.js.map`. The `main.js` file contains all of the application's source code after it has been transpiled to JavaScript, bundled, and [minified](http://bit.ly/2rVdhNk). This allows the application to be served as efficiently as possible. The `main.js.map` file is used to allow developers to view the original source code when debugging the application.
+
+Since all of the files in the `dist` folder are optimized for use in production, deployment only involves copying the contents of the `dist` folder to the location that the web application expects them to be.
+
+While the default `dojo build` command can handle many common scenarios, there are some arguments that can be passed to the command to influence the build process. We will talk about those in the next section.
+
+{% section %}
+
+## Build options
+
+{% task 'Learn how to customize a Dojo 2 build.' %}
+
+The default options for `dojo build` are sufficient for many situations, but there are cases where additional configuration is required.
+
+{% instruction 'Run the `dojo build help` command and examine the result.' %}
+
+There are four groups of command line arguments that are available when using the `dojo build` command.
+
+The first group consists of the `--watch` and `--port` arguments. These are used to start and configure the development server that is designed to host Dojo 2 applications as they are being authored.
+
+The `--help` argument displays the help information for the `dojo build` command and is equivalent to the `dojo build help` command.
+
+{% aside 'I18n in Dojo 2' %}
+Dojo 2 has a complete set of internationalization (i18n) capabilities that are beyond the scope of this tutorial. To learn more about i18n in Dojo 2, refer to the [Internationalization in Dojo 2 tutorial](../comingsoon.html).
+{% endaside %}
+
+The `--locale`, `--supportedLocales`, and `--messageBundles` arguments are used to configure the build's support for internationalization. The `--locale` argument is used to specify which language should be considered the *default*. Whenever a different locale is used that does not have a specific translation, the default locale's translation will be used. The `--supportedLocales` argument is used to specify all of the locales that the application is designed to work with. Each locale that is added here will have its rules for date formatting, currency, etc. added to the build. Finally, the `--messageBundles` argument specifies all of the message bundles, or translation files, that should be included in the build. For more information about creating internationalized applications with Dojo 2, refer to the [Internationalization](../comingsoon.html) article in the reference guide.
```

Comment:
specify all of the locales that the application is designed to work with  -> specify all of the provided locales for the application